corehq/apps/reports/standard/deployments.py

```diff
@@ -543,17 +544,17 @@ def _get_formatted_assigned_location_names(self, user, user_loc_dict):
             loc_name = user_loc_dict.get(loc_id)
             if loc_id == primary_location_id:
                 formatted_loc_names.insert(
-                    0, f'<strong>{loc_name}</strong>'
+                    0, format_html('<strong>{}</strong>', loc_name)
                 )
             else:
-                formatted_loc_names.append(loc_name)
+                formatted_loc_names.append(format_html('{}', loc_name))
 
-        formatted_str = ', '.join(formatted_loc_names[:4])
+        formatted_str = mark_safe(', '.join(formatted_loc_names[:4]))
         html_nodes = [
-            f'<span class="locations-list">{formatted_str}</span>',
+            format_html('<span class="locations-list">{}</span>', formatted_str)
         ]
```

Comment:
I think this `format_html` could and probably should be replaced with [`format_html_join`](https://docs.djangoproject.com/en/5.2/ref/utils/#django.utils.html.format_html_join), which should allow you to remove the `mark_safe` call.